features/step_definitions/shared/workspace.rb

```diff
@@ -7,6 +7,6 @@
   @dir = Dir.mktmpdir
 end
 
-After do |scenario|
+After do |_scenario|
```

Comment:
We can remove `|_scenario|` since we are not using it.

Reply:
I don't remember changing this, might have been rubocop. Will remove.